shell/common/shell.cc

```diff
@@ -1357,4 +1362,36 @@ fml::Status Shell::WaitForFirstFrame(fml::TimeDelta timeout) {
   }
 }
 
+bool Shell::ReloadSystemFonts() {
+  FML_DCHECK(is_setup_);
+  FML_DCHECK(task_runners_.GetPlatformTaskRunner()->RunsTasksOnCurrentThread());
+
+  if (!engine_) {
+    return false;
+  }
+  engine_->GetFontCollection().GetFontCollection()->SetupDefaultFontManager();
+  engine_->GetFontCollection().GetFontCollection()->ClearFontFamilyCache();
+  // After system fonts are reloaded, we send a system channel message
+  // to notify flutter framework.
+  rapidjson::Document document;
+  document.Parse("{}");
+  auto& allocator = document.GetAllocator();
+  rapidjson::Value message_value;
+  message_value.SetString(kFontChange, allocator);
+  document.AddMember(kTypeKey, message_value, allocator);
+
+  rapidjson::StringBuffer buffer;
+  rapidjson::Writer<rapidjson::StringBuffer> writer(buffer);
+  document.Accept(writer);
+  std::string message(buffer.GetString(), buffer.GetSize());
```

Comment:
IIRC you only need `GetString()`, since we should be building rapidjson with std::string support. Am I misremembering?